add mission spcl 97 you have my word to repo

FluffSquirrel opened the pull request:
Mission Pull Request Template

This Pull Request will:

Add the mission you have my word made by me to the repository.

**Mission Creation Checklist:**
- [x] Mission File
- [x] Gearscripts Tested
- [x] Ingame Play Area Markers
- [x] Ingame Briefings to all players
- [x] Side-based briefings  
- [x] Mission .conf file


**Faction(s):**

blufor - police
opfor- the family
indfor - striking workers
civ - hostage factory owner

**Objectives/Win Conditions:**

Not putting in pr description because the objectives for each faction are intentionally withdrawn

**Terrain:**

Eden

**Short mission description:**

<img width="340" height="262" alt="image" src="https://github.com/user-attachments/assets/c9598a6e-d618-4832-8bf0-a135744de57d" />

**Slotting Ratio:**

I think its 6:2:1 + hvt but idk just drop a blufor squad if its low numbers or smthn

**Time limit:**

40m listed but it really just needs to be run based on vibe

**Mission Briefing Screenshot:**
<img width="1560" height="926" alt="image" src="https://github.com/user-attachments/assets/f2dc2ba0-4ea3-4ba8-91c7-cf02fb9b1ae3" />

**Design concept/thoughts:**

Started based on sub rosa but it morphed over time. Again dont want to put much here just ask em in dm if you have questions please

**Other Notes:**

Mentioned it in dev but Id like for the spcl to be listed in AAR and let people sign up for the 4 faction leader roles ahead of time so i can send them info before it starts. That way they can think ahead and not get hit with a wall of info they have to digest and plan with in 5 minutes. If that happens i should be able to explain the mission in a reasonable amount of time before it starts.
Safestart should run for a few minutes, like 3, to let people get oriented. Definitely doesnt need to go any longer than 5m

tanaaKa commented:
Are you still looking to have this played, and if so, how so?

shimavitz commented:
-missing table/floating items in police spawn
-one of the goons can spawn in a garage wall.
-most people including many team leaders don't have maps, mostly talking police

FluffSquirrel commented:
> -missing table/floating items in police spawn
Fixed

> -one of the goons can spawn in a garage wall. 
Moved all spawns to more open areas. Only so much that can be done, especially with indfor until patman adds selecting a spawn radius for the spawnpoints

>-most people including many team leaders don't have maps, mostly talking police
Added to BLUFOR TLs, they were intended to have them. Left the majority without maps though to encourage sticking together and listening to leaders.
